docs/guide/multisite/multisite_configuration.md

```diff
@@ -0,0 +1,193 @@
+# Multisite configuration
+
+You configure the available SiteAccesses and under the `ezplatform.siteaccess` key.
+
+## SiteAccess configuration
+
+``` yaml
+ezplatform:
+    siteaccess:
+        list: [site, event]
+        groups:
+            site_group: [site]
+            event_group: [event]
+        default_siteaccess: site
+        match:
+            URIElement: 1
+```
+
+### SiteAccess groups
+
+`ezplatform.siteaccess.groups` defines which groups SiteAccesses belong to.
+
+``` yaml
+ezplatform:
+    siteaccess:
+        groups:
+            site_group: [site]
+            event_group: [event]
+```
+
+Groups are useful when you want to use common settings for several SiteAccesses and avoid duplicating configuration.
+SiteAccess groups act like regular SiteAccesses as far as configuration is concerned.
+A SiteAccess can be part of several groups. SiteAccess configuration has always precedence over group configuration.
+
+#### `admin` SiteAccess
+
+The predefined `admin` SiteAccess in `admin_group` (configured in `config/packages/ezplatform_admin_ui.yaml`) serves the Back Office.
+Do not remove this group.
+If you need a multisite setup with multiple Back Offices, add any additional administration SiteAccesses to `admin_group`.
+
+In cases where the sites are on separate databases, each needs its own [repository](../configuration.md#configuration-examples)
+(including their own storage and search connection), var dir, [cache pool](../persistence_cache.md#persistence-cache-configuration),
```

Comment:
I think it needs to be stated more clearly, maybe as a next sentence, emphasized as caution/warning that two different SiteAccesses can have different `var_dir` only if they also have different `repository`. 

_This is a flaw in the architecture - `var_dir` should be a `repository`, not SA setting, but we can't change that easily._